City-of-Helsinki/drupal-helfi-rekry
# UHF-6210: Organization taxonomy translations

# [UHF-6210](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6210)
Importing all (Swedish and English) language versions of the Organization taxonomy terms as translations of the Finnish term instead of separate terms.

## What was done
- Implemented a new import subscriber that saves the language versions as translations of the original term
- Added the missing translation imports to be run with cron

## How to install
1. set up REKRY instance
2. checkout this branch: `git checkout UHF-6210_organization_taxonomy_translation`
3. clear cache: `make drush-cr`

## How to test
1. access the application container with `make shell`
2. Import Swedish translations of the taxonomy terms: `drush mim helfi_rekry_organizations:all_sv --update`
3. Import English translations of the taxonomy terms: `drush mim helfi_rekry_organizations:all_en --update`
4. In `/admin/structure/taxonomy/manage/organization/overview`, see the translations of any **Finnish** term and see that the language versions have been imported correctly
